api/db/migrate/20180328223945_add_add_funds_to_groups.rb

```diff
@@ -0,0 +1,5 @@
+class AddAddFundsToGroups < ActiveRecord::Migration
+  def change
+    add_column :groups, :add_funds, :string
```

Comment:
should this be `:string` or `:text` ?   I'm a bit rusty but memory and a quick search suggests `:string` has a length constraint (perhaps 255).

I'll see if I can break it on staging !